Editor/CoreAPI/StateManager.cs

```diff
@@ -39,9 +44,12 @@ private void SetProfile(string profileName)
         {
             if (string.IsNullOrWhiteSpace(profileName)) return;
             _selectedProfile = profileName;
+            var credentials = CoreApi.RetrieveAwsCredentials(profileName);
+            Region = credentials.Region;
             GameLiftWrapper = AmazonGameLiftWrapperFactory.Get(SelectedProfile);
             FleetManager = new GameLiftFleetManager(CoreApi, GameLiftWrapper);
             ComputeManager = new GameLiftComputeManager(CoreApi, GameLiftWrapper);
+            OnProfileSelected?.Invoke();
```

Comment:
If the developer selects the same profile as is currently selected, should we invoke this again? My initial thought is no in which case you can add another check to the if above if they're the same, return without doing anything. If you disagree though, let me know

Reply:
I agree with this